src/cli/index.js

```diff
@@ -2,5 +2,12 @@
  * CLI goes here
  */
 const program = require('commander');
+const packages = require('../../package');
+const startServer = require('../server/httpServer');
 
-console.log('Cli goes here');
+program
```

Comment:
It also could be great to let the user chose a port if the default one is already use

What do you think of a default port of 5000, but a `-p` argument to be able to modify it ? 

like

```
$ someComand -p 7000
```

This will pass the port to the `startServer(port)`

Reply:
Ok i will do that